samples/subsys/fs/README.rst

```diff
@@ -0,0 +1,29 @@
+.. _fat_fs:
+
+FAT Filesystem Sample Application
+###################################
+
+Overview
+********
+
+This sample app demonstrates use of the filesystem API and uses the Fat file
+system driver to mount an SDHC card connected over SPI bus.
+
+Requirements
+************
+
+This project requires a SDHC or microSD card formatted with fat filesystem.
+Please see documentation for the SDHC driver for any current limitations.
```

Comment:
Need a reference (a link) to this documentation, probably this change will suffice:
```
See the :ref:`SDHC_disks` documentation for Zephyr implementation details.
```